src/RefreshManager.m

```diff
@@ -486,6 +486,7 @@ -(void)refreshFeed:(Folder *)folder fromURL:(NSURL *)url withLog:(ActivityItem *
         if (![theLastUpdateString isEqualToString:@""])
         {
             [myRequest addRequestHeader:@"If-Modified-Since" value:theLastUpdateString];
+            [myRequest addRequestHeader:@"A-IM" value:@"feed"];
```

Comment:
That is a good question. 

`A[ccept]-I[nstance]M[anipulation]: gzip` is equivalent to `Accept-Encoding: gzip`, which is already set. `Accept-Encoding` is more widely supported. Theoretically, there shouldn’t be any adverse effects from including it in both places, but it could potentially cause double-gzipping if any part of the delivery chain doesn’t understand “Content-Encoding: gzip”. I’d say there is nothing to gain by including the gzip qualifier in the `A-IM` header as well as in the `Accept-Encoding` header.

So either:
* Merge `Accept-Encoding`’s `gzip` value in to `A-IM` and loose support from a lot of implementations in the process,
* or don’t include `gzip` in the `A-IM` header and leave `Accept-Encoding` as-is.

I hope this helps you decide how you want your implementation to be. I’m happy either way.